tests/core/test_create_cmdline.py

```diff
@@ -272,7 +273,15 @@ def _mock_read_manifest(chip, manifest, clobber=False, clear=False):
 
     for kp, step, index, val in expected_data:
         print("Check", kp, c.schema.get(*kp, step=step, index=index), val)
-        assert c.schema.get(*kp, step=step, index=index) == val
+        stype = c.schema.get(*kp, field='type')
+        if stype in ('file', 'dir'):
+            assert pathlib.Path(c.schema.get(*kp, step=step, index=index)) == pathlib.Path(val)
+        elif stype in ('[file]', '[dir]'):
+            actual_paths = c.schema.get(*kp, step=step, index=index)
+            for i in range(len(val)):
+                assert pathlib.Path(actual_paths[i]) == pathlib.Path(val[i])
+        else:
+            assert c.schema.get(*kp, step=step, index=index) == val
```

Comment:
this should not need to change (that would imply that this change is not backwards compatible)

Reply:
Python cannot guarantee that the exact original string is returned from a cross-platform path conversion, only that the path representation has not changed.

There's no limit to how many strings could refer to the same path. `tapcell.tcl`, `./tapcell.tcl`, `././tapcell.tcl`, `.//.//tapcell.tcl`, ...